### DISCUSSION TITLE
Wiki js version 2.5 me cierra sesion

### DISCUSSION THREAD

**Discussion by eomar23** [requarks/wiki#6411] [Help / Questions]
Buen Dia

Tengo instalado Wiki js versión 2.5 enlazado al Active Directory, después de actualizar el Wiki a la versión 2.5.296 cuando ingreso
con las credenciales a los 2 segundos me Cierra la Sesión, cabe mencionar que antes de la actualización me funciona bien.
Nose si hay que cambiar algún parámetro en la Autentificación.

Alguien que pueda ayudar porfavor.

**Comment by laguilarf** [comment]
Same problem here with version 2.5.296
